index.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html>
+
+<body>
+    <h1>Genotify</h1>
+    <form>
+        Search for:
+        <br>
+        <input type="text" id="query" placeholder="CDK2, cyclin kinase 2, etc">
+    </form>
+    <br>
+    <button id="searchButton" type="button">Search</button>
+    <p id="basics"></p>
+    <br>
+    <p id="summary"></p>
+    <script>
+    (function() {
+        var httpRequest;
+        document.getElementById("searchButton").addEventListener('click', makeQuery);
+
+        function makeQuery() {
+            httpRequest = new XMLHttpRequest();
+            var term = document.getElementById("query").value;
+
+            if (!httpRequest) {
+                alert('Giving up :( Cannot create an XMLHTTP instance');
```

Comment:
Please don't use `alert()`. It provides a poor user experience. Use `console.error(string)` for now. In the future, we can incorporate errors directly into the page.

Reply:
Oh, I think that was a holdover from an example I was looking at. Will do as you suggest.